CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ cmake_minimum_required(VERSION 2.8.12)
 
 include(GNUInstallDirs)
 
+option(CURA_BINARY_DATA_DIRECTORY "Directory to the cura-binary-data repository")
+
+if(NOT CURA_BINARY_DATA_DIRECTORY AND NOT $ENV{CURA_BINARY_DATA_DIRECTORY} EQUAL "")
```

Comment:
It is probably better to use DEFINED $ENV{CURA_BINARY_DATA_DIRECTORY}. In addition, EQUAL does not do string comparison, you need to use STREQUAL for that.